azurerm/resource_arm_firewall_application_rule_collection.go

```diff
@@ -104,6 +104,7 @@ func resourceArmFirewallApplicationRuleCollection() *schema.Resource {
 										ValidateFunc: validation.StringInSlice([]string{
 											string(network.AzureFirewallApplicationRuleProtocolTypeHTTP),
 											string(network.AzureFirewallApplicationRuleProtocolTypeHTTPS),
+											string(network.AzureFirewallApplicationRuleProtocolTypeMssql),
```

Comment:
could we add this new value to [the documentation too](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/firewall_application_rule_collection.html.markdown#L121)?